.github/workflows/test-cm-tutorial-tvm-pip.yml

```diff
@@ -5,7 +5,7 @@ name: CM(CK2) tutorial tvm pip install
 
 on:
   pull_request:
-    branches: [ "test" ]
+    branches: [ "master", "test" ]
```

Comment:
I think it may be important to have at least one test with the TVM backend, because it happens that some changes at other levels of the workflow or its dependencies cause breakdowns in this pipeline. Since CI checks are run in parallel, the difference with those already running will be relatively small, only about 3-5 minutes. Currently we also work on workflow  implementation for other models, and in the future in this test it will probably be possible to use a lighter model that will compile faster (for instance, MobileNet)

Reply:
yes no doubt the test is important. Lets try enabling it and see.